colabs/scikit/decision-trees.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    wandb.sklearn.plot_confusion_matrix(y_test, y_pred, config.labels)
</pre><p>Not sure if we are promoting this api usage. Correct me if I am wrong. Should we use <a href="https://wandb.ai/wandb/plots/reports/Confusion-Matrix--VmlldzozMDg1NTM?galleryTag=visualization" target="_blank">this</a> method instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Was unaware of this feature 😅. Will run next experiments and then revert.